app/assets/javascripts/app/lib/app_post/z_searchable_multiple_select.coffee

```diff
@@ -0,0 +1,43 @@
+class App.SearchableMultipleSelect extends App.SearchableAjaxSelect
+
+  events:
+    'click .selected_item':      'testClick'
+
+  elements:
+    '.items':                     'itemList'
+
+  render: ->
+    @attribute.valueName = ''
+    organizations = {}
+    for value in @attribute.value
+      organizations[value] = App.Organization.findNative(value).name
```

Comment:
This is still a generic component. Please move this code to specific one. Specific components usually have a matching name. Generic ones have a generic name.

Reply:
I have created this component. It's a specific component for my purposes. I don't understand why you ask to move my code from my component. Should I rename it?